scripts/pegasus_rebalance_cluster.sh

```diff
@@ -49,13 +49,6 @@ pwd="$( cd "$( dirname "$0"  )" && pwd )"
 shell_dir="$( cd $pwd/.. && pwd )"
 cd $shell_dir
 
-source ./scripts/minos_common.sh
-find_cluster $cluster
-if [ $? -ne 0 ]; then
-  echo "ERROR: cluster \"$cluster\" not found"
-  exit 1
-fi
-
```

Comment:
why delete？

Reply:
The rebalance script doesn't use any command of minos, only remains the cluster name check. Current scripts who call it all do cluster name check before, deleting it will not affect current scripts. 
`add_ingest_behind_script` call rebalance but won't rely on minos, so I delete it.